src/main/java/com/google/devtools/build/lib/actions/ResourceSet.java

```diff
@@ -135,7 +159,7 @@ public ResourceSet convert(String input) throws OptionsParsingException {
         if (memoryMb <= 0.0 || cpuUsage <= 0.0) {
           throw new OptionsParsingException("All resource values must be positive");
         }
-        return create(memoryMb, cpuUsage, Integer.MAX_VALUE);
+        return create(memoryMb, cpuUsage, null, Integer.MAX_VALUE);
```

Comment:
This could be expanded to also take comma-separated key-value pairs of extra resources. The downside would be that typos aren't easy to check, leaving an opening for confusion.